cmp/grid/GridModel.js

```diff
@@ -695,9 +696,19 @@ export class GridModel {
     autoSizeColumns(colIds = this.getLeafColumns().map(col => col.colId)) {
```

Comment:
(Realize this change is not related to this PR, but I find the expression in the signature too complicated).  We should have colIds default to undefined, and then have the method do this defaulting on the first line.  Also add a docstring.